doc/user-manual/language/copatterns.lagda.rst

```diff
@@ -21,7 +21,9 @@
 Copatterns
 **********
 
-Consider the following record:
+.. note:: If you are looking for information on how to use copatterns with 
+   coinductive records please go to the section on :ref:`coinduction <coinduction>`. 
+   Now consider the following record:
```

Comment:
Er, the "Now consider" should not be part of the note.  Don't you need an empty line after the note?